src/app/rmmapi/messagelist.service.ts

```diff
@@ -146,22 +145,15 @@ export class MessageListService {
                     }
 
                     this.folderListSubject.next(folders);
-                    // Moved from constructor (subscribing to our own
-                    // subjects seems silly)
-                    // Xapian counts:
+                    // Will fallback on the folder counters set above for folders not in the search index
                     this.refreshFolderCounts();
-                    // API counts:
                     this.folders.next(
```

Comment:
can we keep the comments that help indicate which part happens where? this is the most confusing part of this code (some of it does things with indexes, some with REST APIs)

Reply:
Turns out nothing happened here, as this was leftover code :P Dropped it in 09186ed